Updates for duration (#900)
neo4j/docs-cypher: modules/ROOT/pages/values-and-types/temporal.adoc

RETURN datetime('2015-06-24T12:50:35.556+0100') AS theDateTime
----

> [caption=]

Reviewer: What do these do?

Author: they remove the "Table x" enumeration from the result tables in the examples.
they're page-global and i noticed that since there is a table before the examples, the indexes of the example tables are offset by 1 relative to the numbering of the examples.

![Screenshot from 2024-02-29 08-45-26](https://github.com/neo4j/docs-cypher/assets/156673635/675eaaa3-1b9c-4357-9845-774f55aea0ec)

vs

![Screenshot from 2024-02-29 08-44-56](https://github.com/neo4j/docs-cypher/assets/156673635/76f433b3-7412-4d34-b6a7-65cc6a277e36)

Reviewer: Ah ok, TIL! In the interests of cleanliness I wonder if excluding the caption from the first table also makes it un-numbered, so that the later tables numberings get in sync. Just because it's a bit annoying to have `[caption=]` for _each_ table. But if that doesn't work, fair enough!

Author: thought of that too, but then decided to rather get rid of the result table numbering because the examples are already numbered

edit: i guess there are many more occurrences where result tables still have a numbering :thinking: i'm undecided 

edit II: https://docs.asciidoctor.org/asciidoc/latest/tables/turn-off-title-label/
```
:table-caption!:
```
^ that could work. i'll try

yes, that works. it removes table captions page-globally though.

Author: disabled them in both files.

It is also possible to access the smaller (less significant) components of a component group bounded by the largest (most significant) component of the group:

> .Less significant components

Reviewer: Shall we rename these to `nested components` optionally with `first/second order` if needed, and alter the surrounding text accordingly (and the rest of the doc)? I find the current naming/phrasing very hard to grasp.

Author: @JPryce-Aklundh

Reviewer 2: Sounds good to me 👍

Reviewer 2: Sounds good to me 👍

Author: used the first/second order component wording

======

> .Extract the month based components of a `DURATION` value

Reviewer: ```suggestion
.Get the total months of a `DURATION` value
```
Check if it applies to more instances.

Author: had to revisit that one because it's not only about months, but month-based components as defined in the table

6+d|Rows: 1
|===

> `d.quarters` and `d.months` add 4 quarters and 12 months respectively of the year from the query.

Reviewer 2: Is "add" the right word here?

Author: rephrased

> `d.quartersOfYear` is the remaining quarter, counting towrads the next full year. Similarly, `d.monthsOfYear` and `d.monthsOfQuarter` count towards the next full year and quarter respectively.

Reviewer 2: ```suggestion
`d.quartersOfYear` is the remaining quarter, counting towards the next full year. Similarly, `d.monthsOfYear` and `d.monthsOfQuarter` count towards the next full year and quarter respectively.
```

======

> .Extract the day based components of a `DURATION` value

Author: ```suggestion
.Get the total days of a `DURATION` value
```

3+d|Rows: 1
|===

`d.weeks` has a value of 3 because the 25 days from the query are three full weeks (or 21 days). `d.daysOfWeek` are the remaining days, counting towards the next full week.

> See tables _Most significant components_ and _Less significant components in <<cypher-temporal-accessing-components-durations>>.

Reviewer 3: ```suggestion
See tables _Most significant components_ and _Less significant components_ in <<cypher-temporal-accessing-components-durations>>.
```

> See tables _First order `DURATION` components_ and _Second order `DURATION` components_ in <<cypher-temporal-accessing-components-durations>>.

Reviewer 2: I prefer xrefs (easier to spot when broken)

Author: changed to xrefs

======

> .Get the most significant second-based components of a `DURATION` value

Reviewer 3: ```suggestion
.Get the most significant seconds-based components of a `DURATION` value
```
Since you use _days-based_ earlier, this should thus be _seconds-based_.

======

> .Extract the most significant second based components of a `DURATION` value

Reviewer: I don't understand what this sentence means, neither how the example compares to the one after.

Reviewer 3: I don't fully understand what this does, but from an editorial point of view, it should be _second-based_.

Author: ye. having some trouble here as well
d.minutes is 61, so probably it adds the 60 minutes from the hour that was set in the query. likewise for the other return values.
the day is not calculated as 24 hours, because it's not second-based (this again only makes sense after looking at the table in "components of durations" - there are three component groups (months, days, seconds, the latter of which contains hours, minutes, seconds, and so forth)).

there's another table after that one, explaining the less significant components of these groups.

it's... not easy to get as is.

Author: added some explanations to several queries, named the relevant tables and added references to them

6+d|Rows: 1
|===

`d.minutes` is the sum of 60 minutes of the hour and the one minute from the query as both `duration.hours` and `duration.minutes` are both seconds-based components.
Similarly, `d.seconds`, `d.milliseconds`, `d.microseconds` and `d.nanoseconds` are sum values of the relevant seconds-based components from the query.

> `d.hours` doesn't take the day from the query into account because `duration.days` is a days-based component.

Reviewer 2: Just personal preference :) 
```suggestion
`d.hours` does not take the day from the query into account because `duration.days` is a days-based component.
```

6+d|Rows: 1
|===

> `d.minutes` has a value of 61 because `duration.hours` and `duration.minutes` are both second-based components.
> It's the sum of the 60 minutes of the hour and the minute from the query.

Reviewer 3: ```suggestion
`d.minutes` is the sum of 60 minutes of the hour and the one minute from the query as both `duration.hours` and `duration.minutes` are both seconds-based components.
```

> Similarly, `d.seconds`, `d.milliseconds`, `d.microseconds` and `d.nanoseconds` are sum values of the relevant second-based components from the query.

Reviewer 3: ```suggestion
Similarly, `d.seconds`, `d.milliseconds`, `d.microseconds` and `d.nanoseconds` are sum values of the relevant seconds-based components from the query.
```

> `d.hours` doesn't take the day from the query into account because `duration.days` is a day-based component.

Reviewer 3: ```suggestion
`d.hours` doesn't take the day from the query into account because `duration.days` is a days-based component.
```

See table _Most significant components_ in <<cypher-temporal-accessing-components-durations>>.

======

> .Get the less significant second-based components of a `DURATION` value

Reviewer 3: ```suggestion
.Get the less significant seconds-based components of a `DURATION` value
```

======

> .Extract the less significant second based components of a `DURATION` value

Reviewer 3: ```suggestion
.Extract the less significant second-based components of a `DURATION` value
```
As above.

5+d|Rows: 1
|===

The returned values all count towards the next full hour, minute or second respectively.

> For example, `d.microsecondsOfSecond` has a value of 111111 because it's the 111111111 nanoseconds from the query in microseconds (rounded down) but it's not another full second.

Reviewer 2: ```suggestion
For example, `d.microsecondsOfSecond` has a value of `111111` because it is the `111111111` nanoseconds from the query in microseconds (rounded down) but it is not another full second.
```
